### ISSUE TITLE
Locale object

### ISSUE THREAD

**Issue by MerkulovDev** [bleenco/ng2-datepicker#299]
How should looks locale object to work with datepicker?

I want to add my lang (ukrainian). It's aviable in last version of date-fns, but this datepicker doesn't work with it.

So i want to add my own locale file. How should it looks like?

**Comment by jkuri** [comment]
I checked and `date-fns` is the latest version available. You have a clear example in the README how to use it.

**Comment by MerkulovDev** [comment]
@jkuri you have tried last version of datepicker(2.2.2) and last version of date-fns(2.0.0-alpha.7) and languages switches correctly?

**Comment by jkuri** [comment]
we won't use `date-fns` alpha version but a stable release.

**Comment by MerkulovDev** [comment]
@jkuri in stable release there is no ukrainian language.

**Comment by jkuri** [comment]
okay I see.. you can still solve this by manually copying `https://github.com/date-fns/date-fns/tree/v2.0.0-alpha.7/src/locale/ua` folder into your application then import it from there.

**Comment by MerkulovDev** [comment]
i have tried this way and it didn't work for me. i get different errors. I copied folder from date-fns v1. 
i will try one more time and show you an error.

**Comment by jkuri** [comment]
okay. maybe the translations format is not same for v1 and v2 and you'll need to add translations by your own. try copying English example as a case from v1 then manually translate to Ukrainian.

**Comment by MerkulovDev** [comment]
@jkuri i will get an error. one minute. i'm on my way to do this.

**Comment by MerkulovDev** [comment]
`File '/src/typings.d.ts' is not a module.`

**Comment by jkuri** [comment]
that is really weird. do you use `angular-cli` or which tool? can you append me an example app as a `.zip` here (just drag&drop here without `node_modules/` in a comment and I will take a look.

**Comment by MerkulovDev** [comment]
@jkuri  yes, angular-cli. project... i will try to give you needed part. couple minutes. maybe i will solve problem with last error.

**Comment by MerkulovDev** [comment]
solved by adding copy of this module v1.28.1  into my nexus repo. also i've added folder ua into locale.
